core/src/main/resources/schema/app/20200428235034_DISTX-421_add_scalingmode_to_instancegroup.sql

```diff
@@ -0,0 +1,11 @@
+-- // DISTX-421 Add scalingmode column to instancegroup table
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE instancegroup ADD IF NOT EXISTS scalingmode VARCHAR(15) DEFAULT 'UNSPECIFIED';
```

Comment:
I think we usually don't set the default value like this as it blocks the table during an upgrade. @lacikaaa do you have an example for this?

Reply:
yes, you have to add it in separate steps to avoid lock, like this:
https://github.com/hortonworks/cloudbreak/blob/master/freeipa/src/main/resources/schema/app/20200406095012_CB-6385_add_version_to_stack.sql
the result should be the same

Reply:
Thanks. Addressed.